backend/python/migrations/versions/0cc810be5494_add_invited_users_db.py

```diff
@@ -0,0 +1,33 @@
+"""add invited_users db
+
+Revision ID: 0cc810be5494
+Revises: 58313513d17f
+Create Date: 2023-02-21 03:05:30.019336
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0cc810be5494'
+down_revision = '58313513d17f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('invited_users',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('email', sa.String(), nullable=False),
+    sa.Column('role', sa.Enum('Admin', 'Regular Staff', 'Relief Staff', name='roles'), nullable=False),
```

Comment:
Was this migration working for you locally? It fails for me. I'm pretty sure why it's failing is because the roles enum exists already in our database, so a small change to the migration might be needed.
```suggestion
   sa.Column('role', postgresql.ENUM('Admin', 'Regular Staff', 'Relief Staff', name='roles', create_type=False), nullable=False),
```
With a `from sqlalchemy.dialects import postgresql` at the top

Reply:
This basically just lets postgres know not to recreate the enum. [source](https://stackoverflow.com/questions/63461381/how-to-use-an-existing-sqlalchemy-enum-in-an-alembic-migration-postgres)

Reply:
i think it worked for me when i did a flask db downgrade base and then upgrade again, but i can add this in and double check if it works!

Reply:
also so far

> Initial quick review: Great work on this, looks like a lot of logic went into this feature! I haven't looked at the code yet, but had some issues running your branch so I'll leave an initial PR review and we can resolve any issues tgt! A couple questions:
> 
> 1. did the migration run fine for u? I had an issue with it (commented the fix in ur pr)
> 2. I’m able to sign up for an account without an invite link, is this happening for you as well? Can we get someone else to test this?
> 3. We can resend invite links to people who already are in the system. Can we do a quick check for this (lower priority than the other two)?
> 4. How did you implement the 1-hour expiry for the links (so I know what to look for in the PR)?
> 5. Once this PR is merged, does it still make sense to have email verification? Will we need to get rid of this from auth service and Firebase (this might be a lot of work, I'm not sure)?

-> so far i fixed the first three items if you want to take a look! as for the one hour expiry, there isn't a way i specifically implemented it (i just took the text from the verification link email which i think is the one that expires in an hour) - i'll look into how to implement an expiry link because as of right now the link is just to our localhost:3000/login page
-> in terms of email verification, i don't think we'll need it if we have sign in verification because it defeats the purpose to verify twice i think in my opinion, but it could be good to just make sure the account was created properly